Ques10.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Create two classes A, B with default constructors.
+ * Inherit a new class called C from A and create a member B inside C. Don't create a constructor for C
+ * Create an object for class C
+ */
+class A1 
+{
+    int a;
+    A1()
+    {
+        a = 10;
+    }
+}
+class B1
+{
+    int b;
+    B1()
+    {
+        b = 5;
+    }
+}
+class C1 extends A1
+{
+    B1 bobject = new B1();
```

Comment:
follow camelcase for obj names or give meaning ful names

Reply:
Updated